writer/writer_test.go

```diff
@@ -15,17 +15,43 @@
 package writer
 
 import (
+	"fmt"
 	"testing"
 
 	"github.com/mendersoftware/artifacts/metadata"
 	"github.com/stretchr/testify/assert"
 )
 
+func TestMarshallInfo(t *testing.T) {
+	info := metadata.MetadataInfo{
```

Comment:
Will do. I am adding things case by case, and when there is enough to add table tests I am trying to do so. Thanks for review.